test/catch2/README.md

```diff
@@ -0,0 +1,39 @@
+# WiredTiger catch2 framework tests
+
+## Building/running
+
+To build these tests, run CMake with `-DHAVE_UNITTEST=1`, then build as
+usual with Ninja.  To run them, go to your build directory and execute
+`./test/catch2/catch2-unittests`.
+
+To run tests for a specific tag (a.k.a. subsystem), put the tag in square
+brackets, e.g. `[extent_list]`. You can specify multiple tags using commas.
+
+You can also use the `--list-tags` option if you're not sure, or even the
+`--help` flag if you're curious and/or lost. There's also further
+command-line help at
+https://github.com/catchorg/Catch2/blob/devel/docs/command-line.md.
+
+## Contributor's Guide
+There are two categories of tests that exist in the catch2 framework. One category focuses on 
+performing internal tests towards a WiredTiger module and the other category contains any unit test
+that do not fit to a module.
+
+### Modular tests
+Any tests to do with an WiredTiger module will be contained in their own module directory. The 
```

Comment:
Reword: Module specific tests are found under the directory matching the modules name. That directory contains unit tests under unit/ and api contract tests under api/.